docs/source/configuration/advanced_configuration.md

```diff
@@ -95,38 +139,27 @@ CONFIG_LOADER_ARGS = {
-```
-
-`OmegaConfigLoader` can load `YAML` and `JSON` files. Acceptable file extensions are `.yml`, `.yaml`, and `.json`. By default, any configuration files used by the config loaders in Kedro are `.yml` files.
-
-To use `OmegaConfigLoader` in your project, set the `CONFIG_LOADER_CLASS` constant in your [`src/<package_name>/settings.py`](../kedro_project_setup/settings.md):
-
-```python
-from kedro.config import OmegaConfigLoader  # new import
-
-CONFIG_LOADER_CLASS = OmegaConfigLoader
-```
-
 ## Advanced Kedro configuration
 
 This section contains a set of guidance for advanced configuration requirements of standard Kedro projects:
 
-* [How to change which configuration files are loaded](#how-to-change-which-configuration-files-are-loaded)
-* [How to ensure non default configuration files get loaded](#how-to-ensure-non-default-configuration-files-get-loaded)
-* [How to bypass the configuration loading rules](#how-to-bypass-the-configuration-loading-rules)
-* [How to use Jinja2 syntax in configuration](#how-to-use-jinja2-syntax-in-configuration)
-* [How to do templating with the `OmegaConfigLoader`](#how-to-do-templating-with-the-omegaconfigloader)
-* [How to use global variables with the `OmegaConfigLoader`](#how-to-use-global-variables-with-the-omegaconfigloader)
-* [How to use resolvers in the `OmegaConfigLoader`](#how-to-use-resolvers-in-the-omegaconfigloader)
-* [How to load credentials through environment variables](#how-to-load-credentials-through-environment-variables)
+- [Advanced configuration](#advanced-configuration)
+  - [OmegaConfigLoader](#omegaconfigloader)
+    - [Advanced `OmegaConfigLoader` features](#advanced-omegaconfigloader-features)
+  - [TemplatedConfigLoader](#templatedconfigloader)
+    - [Provide template values through globals](#provide-template-values-through-globals)
+  - [Advanced Kedro configuration](#advanced-kedro-configuration)
+    - [How to change which configuration files are loaded](#how-to-change-which-configuration-files-are-loaded)
+    - [How to ensure non default configuration files get loaded](#how-to-ensure-non-default-configuration-files-get-loaded)
+    - [How to bypass the configuration loading rules](#how-to-bypass-the-configuration-loading-rules)
+    - [How to use Jinja2 syntax in configuration](#how-to-use-jinja2-syntax-in-configuration)
+    - [How to do templating with the `OmegaConfigLoader`](#how-to-do-templating-with-the-omegaconfigloader)
+      - [Parameters](#parameters)
+      - [Catalog](#catalog)
+      - [Other configuration files](#other-configuration-files)
+    - [How to use global variables with the `OmegaConfigLoader`](#how-to-use-global-variables-with-the-omegaconfigloader)
+    - [How to use resolvers in the `OmegaConfigLoader`](#how-to-use-resolvers-in-the-omegaconfigloader)
+    - [How to load credentials through environment variables](#how-to-load-credentials-through-environment-variables)
```

Comment:
```suggestion
- [How to change which configuration files are loaded](#how-to-change-which-configuration-files-are-loaded)
- [How to ensure non default configuration files get loaded](#how-to-ensure-non-default-configuration-files-get-loaded)
- [How to bypass the configuration loading rules](#how-to-bypass-the-configuration-loading-rules)
- [How to use Jinja2 syntax in configuration](#how-to-use-jinja2-syntax-in-configuration)
- [How to do templating with the `OmegaConfigLoader`](#how-to-do-templating-with-the-omegaconfigloader)
 - [How to use global variables with the `OmegaConfigLoader`](#how-to-use-global-variables-with-the-omegaconfigloader)
- [How to use resolvers in the `OmegaConfigLoader`](#how-to-use-resolvers-in-the-omegaconfigloader)
- [How to load credentials through environment variables](#how-to-load-credentials-through-environment-variables)
```

Reply:
This also is linking to previous sections that we don't need to mention here.

Reply:
I suspect this is the Markdown VSCode extension? I've observed similar things locally. Maybe the `toctree` directive from Sphinx would be useful here too.

Reply:
It was indeed the extension. Should be fixed now.